stable/coredns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: coredns
-version: 1.7.4
+version: 1.7.5
```

Comment:
Bump MINOR version since this is a new feature (see https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#versioning / semver.org v2)

Reply:
But otherwise this looks good

Reply:
Thanks for the review. Applied the change.